ginivision/src/doc/source/supporting-tablets.rst

```diff
@@ -1,65 +0,0 @@
-Supporting Tablets
-==================
-
-The Gini Vision Library can be used on tablets, too. Some UI elements have been adapted to offer the best user experience for tablet users. Requirements and resources were also adapted.
-
-You may skip to the `Quick Checklist`_ to get an overview of the steps required for supporting tablets.
-
-Extraction Quality Considerations
----------------------------------
-
-We recommend implementing checks on tablet hardware to ensure that devices meet the Gini Vision Libraries minimum recommended hardware specifications.
-
-Many tablets with at least 8MP cameras don't have an LED flash (like the popular Samsung Galaxy Tab S2) therefore we don't require flash for tablets. For this reason the extraction quality on those tablets might be lower compared to smartphones.
```

Comment:
Many tablets with at least 8MP cameras don't have an LED flash (like the popular Samsung Galaxy Tab S2). Therefore, we don't require flash ...